src/resources/pilots.c

```diff
@@ -4,17 +4,20 @@
 void pilot_get_info(pilot *pilot, int id) {
     // TODO read this from MASTER.DAT
     // XXX the colors are eyeballed
+    pilot->sex = 0;
     switch(id) {
         case 0:
+            // CRYSTAL
```

Comment:
```suggestion
        case PILOT_CRYSTAL:
```

Use the enums instead of adding comments, please